### DISCUSSION TITLE
请问如何在文章插入图片时将自动生成的markdown语法种的name去掉？

### DISCUSSION THREAD

**Discussion by miniLQ** [Qexo/Qexo#589] [Q&A] [answered]
现在的
！[https://hexoimg.oss-cn-shanghai.aliy*************1ac5c.png]（https://hexoimg.oss-cn-shanghai.aliy*************1ac5c.png）
期望
！[]（https://hexoimg.oss-cn-shanghai.aliy*************1ac5c.png）

希望默认name为空，如果有需要，自己加，不然每次删好麻烦

**Comment by am-abudu** [comment, marked as answer]
已推送dev分支
